src/charts/stacked-area.js

```diff
@@ -326,18 +329,20 @@ define(function(require){
                 .attr('transform', `translate(${margin.left},${margin.top})`);
 
             container
-              .append('g').classed('x-axis-group', true)
-              .append('g').classed('x axis', true);
+                .append('g').classed('x-axis-group', true)
+                .append('g').classed('x axis', true);
             container.selectAll('.x-axis-group')
-              .append('g').classed('month-axis', true);
+                .append('g').classed('month-axis', true);
             container
-              .append('g').classed('y-axis-group axis', true);
+                .append('g').classed('y-axis-group axis', true);
             container
-              .append('g').classed('grid-lines-group', true);
+                .append('g').classed('grid-lines-group', true);
             container
-              .append('g').classed('chart-group', true);
+                .append('g').classed('y-axis-label', true)
```

Comment:
Just added tabs, the tab width was not consistent compared to the rest of the file and project.

Reply:
Oh, that was intentional for 'append' as it is a operation that changes the operation, and to mark it we use 2 spaces instead of 4.